modules/govuk/manifests/apps/asset_manager.pp

```diff
@@ -43,6 +46,7 @@
   $enabled = true,
   $port = '3037',
   $enable_delayed_job_worker = true,
+  $enable_procfile_worker = true,
```

Comment:
I think you have to make `redis_host` and `redis_port` available via this class, like:

https://github.com/alphagov/govuk-puppet/blob/418a2762da0f48d69e9060f1ac1dd3e588faa04a/modules/govuk/manifests/apps/collections_publisher.pp#L88-L91

Reply:
Does 104031b look correct?

Reply:
I fixed the lint error in 9288014

Reply:
Yes, looks good! Can you add it to the class documentation too?

Reply:
Done in 781b882